index.bs

```diff
@@ -147,7 +152,8 @@ This specification reuses syntax and semantics used in [[!AV1-ISOBMFF]].
 
 <h3 id="image-item">AV1 Image Item</h3>
 
-When an image item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>, it is called an <dfn>AV1 Image Item</dfn>, and shall obey the following constraints:
+When an item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>, it is called an <dfn>AV1 Image Item</dfn>, and shall obey the following constraints:
```

Comment:
Just wanted to confirm that you meant to delete the word "image" from "When an image item is of type ..."

Reply:
Yes, 'av01' implies that it is an image item.